README.md

```diff
@@ -21,9 +21,16 @@ Until we publish (and consume) the HCO and component operators through
 Marketplace|operatorhub.io, this is a means to demonstrate the HCO workflow
 without OLM.
 
+You can either run the following script:
+```bash
+deploy/deploy.sh
+```
+or execute the steps below manually.
```

Comment:
I think it is important to NOT to keep the steps here. If we do, we end up needed to keep to places up to date.